Apps/Sandcastle/gallery/development/Multiple Shadows.html

```diff
@@ -0,0 +1,132 @@
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+
+<script id="cesium_sandcastle_script">
+    function startup(Cesium) {
+        'use strict';
+//Sandcastle_Begin
+
+        var viewer = new Cesium.Viewer('cesiumContainer', {
+            scene3DOnly : true,
+            infoBox : false,
+            selectionIndicator : false,
+            timeline : false
+        });
+        viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
+            url : '//assets.agi.com/stk-terrain/world',
+            requestWaterMask : true,
+            requestVertexNormals : true
+        });
+
+        var scene = viewer.scene;
+        var context = scene.context;
+        var camera = scene.camera;
+
+        scene.debugShowFramesPerSecond = true;
+
+        function addPrimitive(lon, lat, height, heading) {
+            var center = Cesium.Cartesian3.fromRadians(lon, lat, height);
+
+            var pointLightCamera = new Cesium.Camera(scene);
+            pointLightCamera.position = center;
+
+            camera.lookAt(center, new Cesium.Cartesian3(25.0, 25.0, 30.0));
+            camera.lookAtTransform(Cesium.Matrix4.IDENTITY);
+
+            var shadowMap = new Cesium.ShadowMap({
+                context : context,
+                lightCamera : pointLightCamera,
+                isPointLight : true,
+                softShadows : false
+            });
+
+            shadowMap.enabled = true;
+
+            var model = scene.primitives.add(Cesium.Model.fromGltf({
+                url : '../../SampleData/models/ShadowTester/Shadow_Tester_Point.gltf',
+                modelMatrix : Cesium.Transforms.headingPitchRollToFixedFrame(center, heading, 0.0, 0.0)
+            }));
+
+            model.readyPromise.then(function(model) {
+                // Play and loop all animations at half-speed
+                model.activeAnimations.addAll({
+                    speedup : 0.5,
+                    loop : Cesium.ModelAnimationLoop.REPEAT
+                });
+            }).otherwise(function(error) {
+                window.alert(error);
+            });
+
+            return shadowMap;
+        }
+
+        var longitude = -1.3324415110874286;
+        var latitude = 0.6954224325279967;
+        var height = 200.0;
+
+        var shadowMap1 = addPrimitive(longitude + 0.00005, latitude, height, 0.0);
+        var shadowMap2 = addPrimitive(longitude - 0.00005, latitude, height, Math.PI);
+
+        shadowMap1.debugShow = true;
+        shadowMap2.debugShow = false;
+
+        Sandcastle.addToolbarButton('Debug Toggle', function() {
+            shadowMap1.debugShow = !shadowMap1.debugShow;
+            shadowMap2.debugShow = !shadowMap2.debugShow;
+        });
+
+        scene.shadowMap = shadowMap1;
+
+        var CustomPrimitive = function(shadowMap) {
```

Comment:
Might want to comment here that this is a workaround to test multiple shadow maps without Cesium yet supporting multiple lights.